recipes/simdjson/0.4.7/conanfile.py

```diff
@@ -48,11 +43,8 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        if self.settings.compiler.cppstd and \
-           not self.settings.compiler.cppstd in self._supported_cppstd:
-          raise ConanInvalidConfiguration("This library requires c++11 standard or higher."
-                                          " {} required."
-                                          .format(self.settings.compiler.cppstd))
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
 
         if not self._is_supported_compiler():
             raise ConanInvalidConfiguration("This library is tested with a family of recent compilers."
```

Comment:
Could you also update the recipe to not use this pattern please? Allow unknown compilers.

Reply:
I mean: check the version of those compilers, but do not raise if compiler is not listed. Like this: https://github.com/conan-io/conan-center-index/blob/b4c4bea0a3c6cc153e35515ab1c4f25a1eca6df0/recipes/ms-gsl/all/conanfile.py#L49

Reply:
Seems like it was added for a reason, see failed builds. Non-11/14 compilers should be excluded. `check_min_cppstd` is not enough for limiting (see 0.5.0 discussion).
I can allow unknown compilers while restricting known compilers by version. What do you think?